### DISCUSSION TITLE
Is this module alive?

### DISCUSSION THREAD

**Discussion by lna1989** [Diizzayy/nuxt-graphql-client#583] [General]
I noticed minimal activity and an increase in the number of bugs without any action.

There is some uncertainty that it will be possible to work with him on more complex projects. The basic functionality works, but if you dive into the details, then everything is not very good, and it is impossible to get answers for many months.

**Comment by LePtiDev** [comment]
I think not. I have a lot of bugs ont it and no body help me. I don't recommande you to use it.
